components/automate-chef-io/content/docs/node-integrations.md

```diff
@@ -63,12 +63,15 @@ The service makes calls to these API:
 * `STS-GetCallerIdentity`
 * `SEC2-DescribeRegions`
 * `IAM-ListAccountAliases`
+* `IAM-GetAccountSummary`
+* `IAM-ListUsers`
 
 Permissions: You'll need at least a global read permission; `arn:aws:iam::aws:policy/ReadOnlyAccess`
 
 ## AWS Credential-less Scanning with Chef Automate
 
 For users running Chef Automate 2 in EC2, we invite you to try out our "AWS-EC2 Credential-less Scanning"!
+Please note that credential-less scanning is not supported for AWS govcloud.
```

Comment:
Since these are the official docs (I think?), does `govcloud` need to be capitalized any certain way?

Reply:
It looks like [for AWS it's `GovCloud`](https://aws.amazon.com/govcloud-us/)

Reply:
good call! ya!